Resources/public/js/views/ez-contenteditview.js

```diff
@@ -53,11 +53,15 @@ YUI.add('ez-contenteditview', function (Y) {
                     this._setFocus();
                 }
             });
-            this.on('languageCodeChange', function (e) {
+            this.after('languageCodeChange', function (e) {
+                this.get('formView').set('languageCode', this.get('languageCode'));
```

Comment:
same as https://github.com/ezsystems/PlatformUIBundle/pull/484/files#r48230407, use `setter`. you should also update the `actionBar`.